src/Components/WebAssembly/Build/src/targets/Compression.targets

```diff
@@ -0,0 +1,61 @@
+<Project>
+  <PropertyGroup>
+
+    <ResolveCurrentProjectStaticWebAssetsDependsOn>
+      $(ResolveCurrentProjectStaticWebAssetsDependsOn);
+      _CompressBlazorApplicationFiles;
+    </ResolveCurrentProjectStaticWebAssetsDependsOn>
+
+  </PropertyGroup>
+
+  <Target Name="_ResolveAssetsToCompress" AfterTargets="_ResolveBlazorGeneratedAssets">
+
+    <PropertyGroup>
+      <StaticResourcesIntermediateOutputPath>$(IntermediateOutputPath)compressed\</StaticResourcesIntermediateOutputPath>
```

Comment:
If the item groups below all have `_` names, should this property do so too?

Or is it the case that, since the target itself has a `_` name, there's no legit way for someone to hook on more "RunAfterTargets" of their own and hence there's no legit way for external code to observe the existence of this property?

Reply:
Yes, I'll do a cleanup pass.